tests/containers/workbenches/jupyterlab/conftest.py

```
@@ -0,0 +1,2 @@
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)